client/src/Components/PlaceDesigner/PlaceDesignerToolBar.tsx

```diff
@@ -32,18 +32,22 @@ class PlaceDesignerToolBar extends React.Component<PlaceDesignerToolBarProps> {
     getDescription(widgetType: PlaceDesignerToolBarItemType) {
         const { messages, formatMessage } = this.props.intl;
 
-        if (widgetType == 'selection') {
-            return "Режим выделения";
-            // return formatMessage(messages.);
-        } else if (widgetType === 'text') {
-            return formatMessage(messages.widgetBarText);
-        } else if (widgetType === 'run-video') {
-            return formatMessage(messages.widgetBarRunVideo);
-        } else if (widgetType === 'hint') {
-            return formatMessage(messages.widgetBarHint);
+        switch (widgetType) {
+            case 'selection':
```

Comment:
don't change `if` to `switch`, I hate `switch`

Reply:
Why? It's more semantic and readable

Reply:
`if`` is more readable for me